pkgs/development/compilers/gambit/build.nix

```diff
@@ -25,10 +25,11 @@ gccStdenv.mkDerivation rec {
   inherit src version git-version;
   bootstrap = gambit-support.gambit-bootstrap;
 
+  nativeBuildInputs = [ git autoconf ];
```

Comment:
Do we need git here or do we need to fake the information it exctracts?

Reply:
It uses git in the makefile.

Reply:
Important is how. If it uses it to extract version information that will not work most likely.

Reply:
Please see the makefile https://github.com/gambit/gambit/blob/master/makefile.in

Reply:
It creates archives and so on with git.